pom.xml

```diff
@@ -19,19 +19,32 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>33</version>
+    </parent>
 
     <groupId>org.apache.xtable</groupId>
     <artifactId>xtable</artifactId>
-    <name>xtable</name>
+    <name>XTable Parent Project</name>
```

Comment:
It will displays the project name clearly while building the xtable. For Example

```sh
[INFO] XTable Parent Project .............................. SUCCESS [  0.316 s]
[INFO] XTable Api Project ................................. SUCCESS [  0.032 s]
[INFO] XTable Hudi Support Parent Project ................. SUCCESS [  0.025 s]
[INFO] XTable Hudi Support Utils Project .................. SUCCESS [  0.019 s]
[INFO] XTable Core Project ................................ SUCCESS [  0.036 s]
[INFO] XTable Utilities Project ........................... SUCCESS [  0.014 s]
[INFO] XTable Hudi Support Extensions Project ............. SUCCESS [  0.015 s]
[INFO] ------------------------------------------------------------------------
```